### ISSUE TITLE
Accessibility - Collapse accordion example - wrong aria attribute multiselectable

### ISSUE THREAD

**Issue by Lausselloic** [twbs/bootstrap#21635]
Hello,

The accordion sample based on collapse plugin use the aria attribute multiselectable set to true in the documentation. But this accordion is not a multiple open at a time, each new panel opening, close the previous one. From my point of view, this attribute need to be remove until a multi open at a time option will be propose.

http://v4-alpha.getbootstrap.com/components/collapse/#accordion-example

What's your opinion @patrickhlauke ?

**Comment by patrickhlauke** [comment]
agree, good catch

**Comment by jkstermitz** [comment]
Help me understand this please?  I'm trying to use the accordion example to implement an expanding vertical nav, with two menus.  I WANT one menu to collapse when I open the other (and vice versa), and have set it up identical to the example....but of course it does not work as the sample code has "true" set.  

I removed the aria-multiselectable attribute and also tried leaving it in but setting it to "false," but neither seem to affect the behavior (both menus are always independently expandable).  

So, is "multi-selectable="false" supposed to work at all?   How is it that the example functions properly?  Inspecting the code doesn't provide a clue.  It seems the data-parent="#accordion" is what is not working.....

**Comment by mportuga** [comment]
@jkstermitz That is just for accessibility. That attribute tell a blind user what behavior to expect.
